corehq/apps/hqwebapp/utils.py

```diff
@@ -100,9 +100,9 @@ def aliased_language_name(lang_code):
         raise KeyError('Unknown language code %s' % lang_code)
 
 
-def decode_password(obfuscated_password, username=None):
+def decode_password(obfuscated_password):
     if settings.OBFUSCATE_PASSWORD_FOR_NIC_COMPLIANCE:
```

Comment:
Is this a different thing than what is being removed in this PR? If not, are we still doing password obfuscation?

Reply:
Hey @millerdev , as mentioned in the description

> This PR removes any logic for password obfuscation or decoding for the web. Mobile apps would still continue to submit obfuscated password, so decode logic for that needs to be retained

We can't undo the setting because some mobile apps will still submit obfuscated password and would need to be decoded. Does that answer your question?

Reply:
Yes, thanks. Any idea on a timeline for when this can be removed completely?

Reply:
In terms of being sure that no mobile requests would come through with obfuscated code, would need all users to be on new version of the three ICDS apps, which would update the app configuration to tell mobile to not obfuscate the code, which does take an order of months.
Meanwhile,
i think we can remove it from HQ and keep it in commcare-icds, if we also it to extend it using the extension mechanism.